examples/star-wars/package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "apollo-server": "^2.2.0",
     "graphql": "^14.0.2",
-    "nexus": "0.12.0-beta.4",
+    "nexus": "0.12.0-beta.9",
```

Comment:
I wasn't able to use `"nexus": "file:../.."`
We should use `yarn link` on CI, wdyt @Weakky @jasonkuhrt ?